.php-cs-fixer.dist.php

```diff
@@ -9,8 +9,7 @@
         'vendor-bin/',
     ])
     ->ignoreDotFiles(false)
-    ->in(__DIR__)
-    ->name('.php_cs.dist');
```

Comment:
Probably not necessary to explicitly configure the file name here since the configuration file now uses `.php` as file extension.